index.ejs

```diff
@@ -22,8 +22,8 @@
 
         <link type="text/css" rel="stylesheet" href="../api/files/style" />
         
-        <script src="vendor/jquery/3.2.0/dist/jquery.js"></script>
-        <script src="vendor/jquery-migrate/3.0.0/dist/jquery-migrate.min.js"></script>
+        <script src="vendor/jquery/3.6.0/dist/jquery.js"></script>
```

Comment:
@superskip can we update this to use `jquery.min.js` instead of the uncompressed `jquery.js` ?  Is there a reason this was using the uncompressed version before?  (this should help load performance for the Tracker Capture app as well 👍 )